### ISSUE TITLE
Gembird doesn't show up after install

### ISSUE THREAD

**Issue by flo269** [Manuel83/craftbeerpi3#86]
Dear all,

I've installed v3.0 and installed the GembirdUSBActor Plugin from the UI and also installed sispmctl from cli.

I've rebooted and still I can't choose Gembird from the drop down.
Any ideas what I could do?

![screen shot 2017-07-16 at 11 37 30](https://user-images.githubusercontent.com/25147331/28246483-3fbb49ca-6a1b-11e7-8695-d6f433cd3c0a.png)
![screen shot 2017-07-16 at 11 37 13](https://user-images.githubusercontent.com/25147331/28246485-41f46762-6a1b-11e7-9476-6386e239ab88.png)

**Comment by carlallen** [comment]
Make sure you are running the latest version of cbpi3. In the System->System menu, click Activate next to master branch and then pull the update.

**Comment by flo269** [comment]
I'm quite sure that I'm on current master now. Pulled, rebooted and still can't select Gembird.

![screen shot 2017-07-17 at 11 51 23](https://user-images.githubusercontent.com/25147331/28263228-47c73d1a-6ae6-11e7-9630-1861304443a2.png)

**Comment by flo269** [comment]
It looks like the repo https://github.com/Manuel83/cbpi-GembridUSB is empty. Or am I mistaking?

**Comment by jalim** [comment]
You're right, Manuel may just be using it as a placeholder or it is a mistake.  Maybe raise an issue in that repository.

**Comment by flo269** [comment]
ok, will do. thanks!

**Comment by Redsak** [comment]
having the same problem here. Any Suggestions so far?
